azurelinuxagent/ga/exthandlers.py

```diff
@@ -76,6 +76,11 @@
 
 NUMBER_OF_DOWNLOAD_RETRIES = 5
 
+DISABLE_FAILED = "DISABLE_FAILED"
```

Comment:
These strings seem generic and can potentially collide with existing environment variables. Can we add something like a `WALinuxAgent` prefix to them so the chance of interfering with customer env variables is lower?

Reply:
Good point but we're only setting the environment variables for that particular process and not the systemwide environment variables. That being said we do pass `os.environ` to the process too which might have the same variables set by the customer. The chance is low of a conflict but definitely can happen. 

How about GUEST_AGENT_DISABLE_FAILED as we want the variable names to be generic for both our agents?

Reply:
Maybe add "AZURE" somewhere there?

Reply:
sounds good, I'll add AZURE_GUEST_AGENT prefix to all the variables (DISABLE_FAILED, UNINSTALL_FAILED, EXTENSION_PATH and EXTENSION_VERSION